.github/CODEOWNERS

```diff
@@ -121,6 +121,7 @@ apps/ts-minbar-test-react @microsoft/fluentui-react-build
 apps/ts-minbar-test-react-components @microsoft/fluentui-react-build
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=31254&commitId=792d4cc8a4be045f4d1b312810f94a65d6116bf2&env=prod&clientType=FLUENTUI)


  <details><summary><b>Checkbox</b> 4 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Checkbox.Checkbox Example.Focuses checkbox.chromium.png|14| Changed|
|Checkbox.Checkbox Example Disabled.chromium.png|3| Changed|
|Checkbox.Checkbox Example.Checks checkbox.chromium.png|14| Changed|
|Checkbox.Checkbox Example Checked.chromium.png|11| Changed|

</details>


  <details><summary><b>Popup</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Popup.Popup Example Auto Size.chromium.png|11| Changed|

</details>


  <details><summary><b>Tree</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Tree.Tree Multiselect Example Teams Dark.chromium.png|3| Changed|
|Tree.Tree Multiselect Example.default selected states.chromium.png|32| Changed|
|Tree.Tree Multiselect Example Teams Dark.default selected states.chromium.png|23| Changed|
|Tree.Tree Multiselect Example.chromium.png|5| Changed|
|Tree.Tree Multiselect Example Teams.default selected states.chromium.png|32| Changed|
|Tree.Tree Multiselect Example Teams Dark.first expanded, not custom checkbox visible.chromium.png|7| Changed|
|Tree.Tree Multiselect Example.first expanded, not custom checkbox visible.chromium.png|10| Changed|
|Tree.Tree Multiselect Example Teams High Contrast.default selected states.chromium.png|40| Changed|
|Tree.Tree Multiselect Example Teams.chromium.png|5| Changed|
|Tree.Tree Multiselect Example Teams.first expanded, not custom checkbox visible.chromium.png|10| Changed|
|Tree.Tree Multiselect Example Teams High Contrast.first expanded, not custom checkbox visible.chromium.png|11| Changed|
|Tree.Tree Multiselect Example Teams High Contrast.chromium.png|6| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>